pkg/secrets/secrets.go

```diff
@@ -117,7 +117,12 @@ func getMounts(filePath string) []string {
 	}
 	var mounts []string
 	for scanner.Scan() {
-		mounts = append(mounts, scanner.Text())
+		if strings.HasPrefix(scanner.Text(), "/") {
```

Comment:
Does scanner chop off preceding white characters?   Should we run this through `strings.TrimSpace()` or just assume they'll not space a line over?

Reply:
It does not strip any whitespace, so I will add a `strings.TrimSpace()`. :+1: